assets/dev/scss/admin/_role-manager.scss

```diff
@@ -49,6 +49,7 @@
 			.elementor-role-go-pro {
 				display: flex;
 				align-items: center;
+				padding: 1em 0 0;
```

Comment:
New json upload notice in Role Manager was added yesterday without any padding between it and the promotion.